services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java

```diff
@@ -112,7 +111,7 @@ public Object getConfig() {
             .path(AuthorizationEndpoint.class, "authorizeDevice").path(DeviceEndpoint.class, "handleDeviceRequest")
             .build(realm.getName(), OIDCLoginProtocol.LOGIN_PROTOCOL).toString());
         URI jwksUri = backendUriBuilder.clone().path(OIDCLoginProtocolService.class, "certs").build(realm.getName(),
-            OIDCLoginProtocol.LOGIN_PROTOCOL);
+                                                                                                    OIDCLoginProtocol.LOGIN_PROTOCOL);
```

Comment:
Minor: there are some changes in the formatting, which are unrelated to this PR. Will it be ok to remove them? Ideally please keep jut new stuff related to OIDCWellKnownProvider, but will be nice to remove all the other things...

Reply:
@mposolda I see, I will find and remove these points that are not related to this PR.

Reply:
Thanks